sqldelight-gradle-plugin/src/test/fixtures/unnecessary-current-index/expected/com/sample/TestModel.java

```diff
@@ -0,0 +1,103 @@
+package com.sample;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import com.squareup.sqldelight.RowMapper;
+import com.squareup.sqldelight.SqlDelightStatement;
+import java.lang.Deprecated;
+import java.lang.Override;
+import java.lang.String;
+import java.lang.StringBuilder;
+import java.util.Collections;
+
+public interface TestModel {
```

Comment:
Do we still want to have this test case? There's already a lot of cases where the this case is covered.